docs/src/modules/spring/pages/actions-as-controller.adoc

```diff
@@ -1 +1,112 @@
 = Actions as Controllers
+
+Actions can be used to implement link:https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller[MVC Controllers] by
+acting as the external interface of a service, receiving requests, operating over the requests values and forwarding the call
+to other components in the same service.
+
+To illustrate how you can use an Action as a Controller, we will build on top of the xref:structuring-your-application.adoc[Value Entity Shopping Cart] example, adding a new Action to the existing shopping cart service.
+
+== Forwarding Commands
+
+The `forward` effect allows us to transform or further validate an incoming request before passing it on to another
+component and have the response message directly passed back to the client making the request. The response from the
+forwarded operation must have the same response type as the original command.
```

Comment:
I understand this is the same text as the original. I struggled a bit to understand it so here is a suggestion. 
`The `forward` effect allows you to call another
component and have the response message passed back to the method forwarding.  The response from the
forwarded operation must have the same response type as the original method. You can transform or further validate an incoming request before passing it on to another component`

I'm not sure we should have different info than the java docs but at the same time I think is a good opportunity to improve the docs. You may want to check with Renato if you are interested in doing such a change.

Reply:
> > return the response message to the forwarding method before passing it back to the client making the request
> 
> Don't think this is accurate actually.. because the forward doesn't allow you to transform the response, so effectively the response is passed directly to the client making the request and not to the forwarding method.

Good point. I guess that depends on what before means here. Before in space or in time? (we are doing philosophy now:) Meaning space as in the code, and time as in runtime. I meant in time but I see now it can be interpreted as in the code. In any case, it was a mild suggestion, I guess the original text is just okay the way it is.

Reply:
Oh I see, makes sense. :) 

Yeah, I couldn't improve it much further I think.. Maybe we can revisit in the future (famous last words).

Reply:
hehehe okay, let's leave it like it is then.